native~/Runtime/src/Cesium3DTilesetImpl.cpp

```diff
@@ -326,6 +326,14 @@ void Cesium3DTilesetImpl::FocusTileset(
 #endif
 }
 
+float Cesium3DTilesetImpl::ComputeLoadProgress(
+    const DotNet::CesiumForUnity::Cesium3DTileset& tileset) {
+  if (getTileset() == nullptr) {
+    return 0;
+  }
+  return getTileset()->computeLoadProgress();
+}
+
```

Comment:
I think generally, as a helper function, this is fine. 

But I'm wondering if it's more something that the caller function should implement, and we should leave out of this code base. Here's why:
* The caller already has access to getTileset()
* This function is small and trivial to implement
* This function is implemented in a specific way. It's interpreting a nullptr tileset as 0 progress. The calling code might want this state handled in a different way.

Open for discussion for sure. I lean towards "less is more". Less to maintain, less to go wrong, etc.

Reply:
As Joseph mentioned offline, this functionality is trivial for us to implement in our C++ code using Reinterop, but it's impossible for users to implement themselves from C# (or even from C++ without modifying the plugin itself). So it's definitely valuable and needed.